Compiler/BackEnd/CommonSubExpression.mo

```diff
@@ -1212,7 +1212,7 @@ algorithm
         //eqAtts := List.threadMap(List.fill(false, arrayLength(m)), List.fill("", arrayLength(m)), Util.makeTuple);
         //BackendDump.dumpBipartiteGraphStrongComponent2(pathVars, eqs, m, varAtts, eqAtts, "shortenPaths"+stringDelimitList(List.map(partition,intString),"_"));
 
-     for varIdx in List.intRange(arrayLength(mT)) loop
+     for varIdx in 1:arrayLength(mT) loop
```

Comment:
@sjoelund @adrpo
This change is correct, but OMC generates weird code from it (or better from the line below). The result of arrayGet() is wrongly mmc_unbox_integer()ed. Looks like a bug in OMC. Shall I create a ticket?
```
-              _varIdx = arrayGet(_pathVarIdxMap, mmc_unbox_integer(_varIdx));
+              _varIdx = mmc_unbox_integer(arrayGet(_pathVarIdxMap, (modelica_integer)_varIdx));
```